inyoka_theme_default/locale/de_CH/LC_MESSAGES/django.po

```diff
@@ -659,6 +435,15 @@ msgstr ""
 msgid "Browse pastes"
 msgstr ""
 
+msgid ""
+"But the article itself is not listed as an attachment, perhaps\n"
+"          these pages are using the article mistakenly or an old revision\n"
```

Comment:
It is better than before, I played a little with the bable config but that only fixed some of the IDs. So I would suggest to accept the PR and wait.